README.md

```diff
@@ -1,98 +1,115 @@
+
+This image inherits from the [thmhoag/steamcmd](https://github.com/thmhoag/steamcmd) image to include the latest version of `steamcmd`.
+
+For more information on `arkmanager`, see the repo here: https://github.com/FezVrasta/ark-server-tools
+
+### Features
+* Automated install (pull the image and run, no additional commands necessary)
+* Configuration via Environment Variables
+* Easy crontab manipulation for automated backups, updates, daily restarts, weekly dino wipes, etc
+* Simple volume structure for server files, config, logs, backups, etc
+* Inherently includes all features present in `arkmanager`
+
+### Tags
+| Tag | Description |
+|--|--|
+| latest | most recent build from the master branch |
+| x.x.x (semver) | release builds |
+
+## Usage
+
+### Installing the image
+
+Pull the latest (or any other desired version):
+```bash
+docker pull thmhoag/arkserver:latest
+```
+
+### Running the server
+
+To run a generic server with no configuration modifications:
+```bash
+$ docker run -d \
+    -v steam:/home/steam/.steam/steamapps \  # mounted so that workshop (mod) downloads are persisted
+    -v ark:/ark \  # mounted as the directory to contain the server/backup/log/config files
+    -p 27015:27015 -p 27015:27015/udp \  # steam query port
+    -p 7778:7778 -p 7778:7778/udp \  # gameserver port
+    -p 7777:7777 -p 7777:7777/udp \ # gameserver port
+    thmhoag/arkserver
+```
+
+If the exposed ports are modified (in the case of multiple containers/servers on the same host) the `arkmanager` config will need to be modified to reflect the change as well. This is required so that `arkmanager` can properly check the server status and so that the ARK server itself can properly publish its IP address and query port to steam.
+
+## Environment Variables
+
+A set of required environment variables have default values provided as part of the image:
+
+| Variable | Value | Description |
+| - | - | - |
+| am_ark_SessionName | `Ark Server` | Server name as it will show on the steam server list |
+| am_serverMap | `TheIsland` | Game map to load |
+| am_ark_ServerAdminPassword | `k3yb04rdc4t` | Admin password to be used via ingame console or RCON |
+| am_ark_MaxPlayers | `70` | Max concurrent players in the game |
+| am_ark_QueryPort | `27015` | Steam query port (allows the server to show up on the steam list) |
+| am_ark_Port | `7778` | Game server port (allows clients to connect to the server) |
+| am_ark_RCONPort | `32330` | RCON port |
+| am_arkwarnminutes | `15` | Number of minutes to wait/warn players before updating/restarting |
+| am_arkflag_crossplay | `false` | Allow crossyplay with Players on Epic |
+| ARKCLUSTER | `false` | Symlinks `ShooterGame/Saved/clusters` to the `/arkclusters` volume |
```

Comment:
not a symlink anymore, right?

Reply:
Hmm, seem to have missed one of the last commits... fixed